deepchem/models/tensorgraph/tests/test_graph_layers.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import tensorflow as tf
+import six
+
+from deepchem.data import NumpyDataset, pad_features
+from deepchem.feat.mol_graphs import ConvMol
+from deepchem.metrics import to_one_hot
+from deepchem.models.tensorgraph.graph_layers import WeaveGather, DTNNEmbedding, DTNNStep, DTNNGather, DAGLayer, DAGGather, DTNNExtract, MessagePassing, SetGather, WeaveLayer, EdgeNetwork, GatedRecurrentUnit
+
+from tensorflow.python.framework import test_util
+
+
+class TestGraphLayers(test_util.TensorFlowTestCase):
+    def Weavegather(self):
```

Comment:
Rename the method to `test_weave_gather`.  nosetest only treats a method as a test if its name begins with "test".